src/main/java/com/microsoft/azure/spring/data/cosmosdb/repository/DocumentDbRepository.java

```diff
@@ -26,5 +26,7 @@
     Observable<T> deleteAllAsync();
 
     Observable<Page<T>> findAllAsync(Pageable pageable);
+
+    Observable<Long> countAllAsync();
```

Comment:
I'm unsure what's the convention for other spring project. My understanding is that method returning `Observerable` might have suffix `Rx` or `Reactive`. `Async` is for methods returning `CompletableFuture`

Reply:
OK, will check list and if necessary, renaming all `-Async` methods to a more proper suffix if exists.